db/change_cache_test.go

```diff
@@ -220,7 +220,7 @@ func WriteDirect(db *Database, channelArray []string, sequence uint64) {
 }
 
 func WriteUserDirect(db *Database, username string, sequence uint64) {
-	docId := fmt.Sprintf("_sync:user:%v", username)
+	docId := fmt.Sprintf(base.UserPrefix+"%v", username)
```

Comment:
Can replace Sprintf with the faster inline concatenation.

```suggestion
	docId := base.UserPrefix + username
```